pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -43,6 +43,15 @@ type KeycloakAPIRealm struct {
 	// A set of Event Listeners.
 	// +optional
 	EventsListeners []string `json:"eventsListeners,omitempty"`
+	// Enable events recording
+	// +optional
+	EventsEnabled *bool `json:"eventsEnabled,omitempty"`
```

Comment:
In this particular case, we use pointers since we want to enable this by default (and with values, we would get `false` here if a user hasn't specified anything). 

Perhaps we could use `// +kubebuilder:validation:Default=true` ? I think the shortcut should also work: `+kubebuilder:default=true`

See [this Pull Request](https://github.com/kubernetes-sigs/controller-tools/pull/323/files#diff-862d6e06a6705fcf1b7782d96f1e003aR98) for more info.

Reply:
Interesting, didn't know about that. Will update the PR shortly.

Reply:
@slaskawi @davidffrench As discussed, kubebuilder annotations do not seem to be supported in any operator-sdk release yet (but there seem to be plans). Leaving the code as is for now (because not sure if we should add a default value annotation for a pointer).

Reply:
Thanks @pb82 . This was working for me before, but since the default field is only supported in CustomResourceDefinition v1, it was possibly removed from the CRD generation for now.

Reply:
That's very interesting @davidffrench! Can we somehow ask `Operator SDK generate`  to produce CRD v1 for us? From our perspective - that would be a desirable move!

Reply:
v1 is only available from k8s 1.16/OpenShift 4.3 . MY assumption is it is on their radar and will be available in a future release. It is already supported in controller-tools which does the actual generation.

Reply:
@pb82 @davidffrench Ok, that sounds good to me. @pb82 May I ask for one additional, tiny change - would you mind to add a `//TODO` there saying that we may want to change it to values (instead of pointers) some day? Apart from that - it looks good to me.

Reply:
@slaskawi Sure, done